apps/condo/domains/acquiring/constants/context.js

```diff
@@ -10,10 +10,20 @@ const CONTEXT_STATUSES = [
     CONTEXT_FINISHED_STATUS,
 ]
 
+const VAT_OPTIONS = [0, 10, 20]
+
+const TAX_REGIME_GENEGAL = 'general'
+const TAX_REGIME_SIMPLE = 'simple'
+const TAX_REGIMES = [TAX_REGIME_GENEGAL, TAX_REGIME_SIMPLE]
+
 module.exports = {
     CONTEXT_IN_PROGRESS_STATUS,
     CONTEXT_VERIFICATION_STATUS,
     CONTEXT_ERROR_STATUS,
     CONTEXT_FINISHED_STATUS,
     CONTEXT_STATUSES,
-}
\ No newline at end of file
+    VAT_OPTIONS,
+    TAX_REGIME_GENEGAL,
+    TAX_REGIME_SIMPLE,
+    TAX_REGIMES,
```

Comment:
Is tax regime a term, or just a translation?

Reply:
What do you mean? These are values to select by user. These values have translations: `pages.condo.marketplace.settings.requisites.radio.tax.`

Reply:
I meant if `regime` a translation or dictionary term. Since i think its important for fields like this to be a term.